packages/flatbread/content/authors/me.md

```diff
@@ -8,7 +8,7 @@ enjoys:
   - making this
 date_joined: 2021-02-25T16:41:59.558Z
 skills:
-  sitting: 204
+  sitting: 69
   breathing: 7.07
   liquid_consumption: 100
```

Comment:
```suggestion
  sitting: 69
  breathing: 7.07
  liquid_consumption: 420
```

Reply:
lmao. this was by coincidence! the mutation test mutates sitting with math.random